GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -431,6 +431,7 @@ private void DiffContextMenu_Opening(object sender, CancelEventArgs e)
             blameToolStripMenuItem.Enabled = _revisionDiffController.ShouldShowMenuBlame(selectionInfo);
             resetFileToToolStripMenuItem.Enabled = _revisionDiffController.ShouldShowResetFileMenus(selectionInfo);
 
+            diffDeleteFileToolStripMenuItem.Text = ResourceManager.Strings.GetDeleteFile(selectionInfo.IsSingleGitItemSelected ? 1 : 2);
```

Comment:
There are languages with different declension for different numbers, e.g. Russian.
So better turn `IsSingleGitItemSelected` into `SelectedGitItemCount`.
Or stay with "Delete File(s)".
```suggestion
            diffDeleteFileToolStripMenuItem.Text = ResourceManager.Strings.GetDeleteFile(selectionInfo.SelectedGitItemCount);
```